test/fixtures/users.yml

```diff
@@ -1,9 +1,31 @@
-# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+admin:
```

Comment:
very nice to see tests,
but fixtures are evil

for such purpose we usually using 
``` https://github.com/thoughtbot/factory_girl ```  

let's add it to the project.  ??
@serhiym @Kmalynss